app/models/comment.rb

```diff
@@ -1,3 +1,7 @@
 class Comment < ApplicationRecord
+  validates :message,
+    length: {maximum: 1500},
+    presence: true
   belongs_to :user_challenge
+  belongs_to :user
```

Comment:
wanted to confirm whether user needed to be added to comment

Reply:
I think it will as comments could come from a mentor or a mentee, where as its fine for notes to just belong to a user challnege